populus/compilation/backends/base.py

```diff
@@ -1,16 +1,39 @@
 import logging
+import os
 
+from eth_utils import (
+    to_tuple,
+)
 from populus.utils.module_loading import (
     get_import_path,
 )
+from populus.utils.filesystem import (
+    recursive_find_files,
+)
 
 
 class BaseCompilerBackend(object):
     compiler_settings = None
+    project_source_extensions = None
+    test_source_extensions = None
```

Comment:
`_glob` would be a more semantically correct suffix for these properties.

Reply:
:+1: